charts/ingress-nginx/templates/controller-daemonset.yaml

```
@@ -62,7 +62,7 @@ spec:
     {{- end }}
     {{- end }}
       containers:
-        - name: controller
+        - name: {{ .Values.controller.name }}
```

Q for reviewers - this is set to "controller" in [the values](https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/values.yaml#L11), but perhaps we should add a default value here, just in case someone overrides it with an empty string?
```suggestion
        - name: {{ .Values.controller.name | default "controller" }}
```